frontend/src/bundles/common/helpers/find-icon.tsx

```diff
@@ -0,0 +1,8 @@
+import { type IconProp } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const findIcon = (iconKey: string): JSX.Element | undefined => {
```

Comment:
It's a component. So its name should not start with a verb. Rename it to something like `Icon`